src/index.js

```diff
@@ -7,6 +7,9 @@ export { default as Divider } from './Divider'
 export { default as GreenButton } from './GreenButton'
 export { default as RedButton } from './RedButton'
 export { default as OutlineButton } from './OutlineButton'
+export { default as Label } from './Label'
```

Comment:
We were starting to keep these in alphabetical order to help with merge conflicts, but think we fell off track a little